common-ui/src/main/res/values/attrs.xml

```diff
@@ -94,4 +94,5 @@
     <attr name="dialogPrimaryButtonColor" format="color" />
     <attr name="dialogSecondaryButtonColor" format="color" />
 
+    <attr name="overflowMenuDividerColor" format="color" />
```

Comment:
as discussed, I'll create a custom view for this so we don't need to keep adding colors / styles